openstf/pipeline/train_model_sklearn.py

```diff
@@ -20,11 +19,8 @@
 OLD_MODEL_PATH = '.'
 
 
-def train_model_pipeline(pj: dict, check_old_model_age: bool = True,
+def train_model_pipeline(pj: dict, input_data, check_old_model_age: bool = True,
```

Comment:
If you use type hinting I think it makes sense to do it for all arguments, i.e. input_data does not have a datatype now.

Reply:
I added pd.DataFrame for the type of input_data. I'm not sure it's a valid type though, as it's not colored like dict or bool (I added import pandas as pd too).

Reply:
I think that should work. My editor does color it properly: 
![image](https://user-images.githubusercontent.com/6715707/118000680-54663880-b346-11eb-8ce9-59e131fd37bd.png)